packages/ui-stencil/src/components.d.ts

```diff
@@ -58,7 +58,7 @@ export namespace Components {
     }
     interface OramaChatBox {
         "autoFocus": boolean;
-        "clientInstance"?: OramaClient;
+        "clientInstance"?: OramaClient | Orama<unknown>;
```

Comment:
Shouldn't we be using OramaSwtich?

Reply:
The `clientInstance` coming as a prop should be an `OramaClient | AnyOrama.`
The Switch is instantiated inside the ChatService / SearchService.
Ex. [ChatService](https://github.com/oramasearch/orama-ui-components/pull/39/files#diff-618f2dd4aaac9973128dd2a0732ba3da71bc1586a01f5c9d8851714443c5957cR13)
[SearchService](https://github.com/oramasearch/orama-ui-components/pull/39/files#diff-6f968c09e991bb7dc36b60a61983e9fa98f9b24facf7b8a3b617bd5223c30062R24)